tools/walletextension/wallet_extension.go

```diff
@@ -269,18 +284,19 @@ func (we *WalletExtension) handleSubmitViewingKey(resp http.ResponseWriter, req
 		return
 	}
 
-	// We drop the leading "0x", and transform the V from 27/28 to 0/1.
```

Comment:
Half of the comment was moved further down, to where the transformation was actually performed.